File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car{
    Map<String, Map<Integer, String>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberVan(String license, Account driver){
        super(license, driver);
    }

    @Override
    public void setPassenger(Integer passenger) {
        if(passenger >= 1 && passenger <= 6){
            this.passenger = passenger;
        }else if(passenger > 6){
            mensajeError = "No puedes subir mas de 4 pasajeros";
        }else if(passenger < 1){
            mensajeError = "El minimo de pasajeros es 1";
        }
    }
}

File: PHP/UberVan.php
<?php
require_once("Car.php");
    class UberVan extends Car{
        private $typeCarAccepted;
        private $seatsMaterial;

        public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial)
        {
            parent::__construct($license, $driver);
            $this->typeCarAccepted = $typeCarAccepted;
            $this->seatsMaterial = $seatsMaterial;
        }

        public function setPassenger($passengers)
        {
            if($passengers >=1 && $passengers <= 6){
                $this->passengers = $passengers;
            }else{
                echo "Necesias asignar unicamente de 1 a 6 pasajeros";
            }            
        }
    }
?>